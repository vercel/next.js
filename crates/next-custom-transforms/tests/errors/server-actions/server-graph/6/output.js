/* __next_internal_action_entry_do_not_use__ {"00c18c215a6b7cdc64bf709f3a714ffdef1bf9651d":"default"} */ import { registerServerReference } from "private-next-rsc-server-reference";
import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
export default /*#__TURBOPACK_DISABLE_EXPORT_MERGING__*/ $$RSC_SERVER_ACTION_0 = ()=>{};
var $$RSC_SERVER_ACTION_0;
Object.defineProperty($$RSC_SERVER_ACTION_0, "name", {
    "value": "default",
    "writable": false
});
import { ensureServerEntryExports } from "private-next-rsc-action-validate";
ensureServerEntryExports([
    $$RSC_SERVER_ACTION_0
]);
registerServerReference($$RSC_SERVER_ACTION_0, "00c18c215a6b7cdc64bf709f3a714ffdef1bf9651d", null);
