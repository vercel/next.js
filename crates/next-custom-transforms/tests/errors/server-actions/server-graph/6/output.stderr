  x Server Actions must be async functions.
  | 
   ,-[input.js:3:1]
 2 | 
 3 | export default () => {}
   :                ^^^^^^^^
   `----
